src/components/TicketsTable.tsx

```diff
@@ -52,6 +52,13 @@ function TeamMemberView({ id, team_member }) {
     </Link>
   );
 }
+function UrgentView({ id, urgent }) {
+  return (
+    <Link className="text-gray-900" href={`?ticket=${id}`} key={id}>
+      {urgent ? 'Urgent' : ''}
```

Comment:
@AdelBeit you may want to change this!! I didn't know what the correct styling was, so I just put in a place holder. Should be an easy change

Reply:
Since the urgent color should indicate not only the word 'urgent' but the whole row will be color red if it's an urgent case/ticket, since there is an urgent flag, we can just use that to render the whole row red. I can do it if it's posing a minor challenge.

Reply:
@tristan-88 maybe we push this PR as is then you can create another PR with the correct styling?

Reply:
yes, i looked into it, the rowClassName prop on data table should do the trick, @tristan-88 feel free to implement it